### ISSUE TITLE
Pop-up menu takes a long time to close

### ISSUE THREAD

**Issue by AleksandrRybalt** [bgrabitmap/lazpaint#363]
**1. Summary:**
Pop-up menu takes a long time to close.
**2. Description:**
Pop-up menu takes a long time to close if you move the cursor away.
**3. Environment:** 
Windows 10 Pro Build 19043.1288
LazPaint 7.1.6
**4. Steps to reproduce**
Hover over the icon with a pop-up window;
Move the cursor to a different location;
Watch the pop-up.
**5. Expected result**
Instantly close the pop-up.
**6. Actual result**
Pop-up closes in about 2 seconds.
**7. Attachment:** 
https://user-images.githubusercontent.com/89531854/139294587-09bdcd74-e21c-497e-a2eb-259a5dc75446.mp4

**Comment by circular17** [comment]
Yes, this is to avoid it closing it too fast if for some reason the mouse is not considered on the panel.

Could be improved by measuring the distance of the mouse.
